packages/core/ios/RNSentry.mm

```diff
@@ -467,7 +467,11 @@ - (void)stopObserving
             contexts[@"os"] = os;
         }
 
-        NSString *releaseName = [SentrySDK options].releaseName;
+#if CROSS_PLATFORM_TEST
+        NSString *releaseName = SentrySDKInternal.options.releaseName;
+#else
+              NSString *releaseName = [SentrySDK options].releaseName;
```

Comment:
nit: I think we should align this on the left though the linter is not complaining.
```suggestion
         NSString *releaseName = [SentrySDK options].releaseName;
```

Reply:
I applied the suggestion, if it breaks, lets revert it.